sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1324,7 +1330,7 @@
         </property>
       </activation>
       <properties>
-        <parallelizeTests>false</parallelizeTests>
+        <parallelizeTests>${parallelizeLiveTests}</parallelizeTests>
```

Comment:
This is RECORD profile but the property name is `parallelizeLiveTests`. Should we have different property names for LIVE, RECORD and PLAYBACK modes or should we call this property name `parallelizeTests` if we want to use the same value for all three?

Reply:
This configuration change is making is so there are two configuration for test parallelization, one for PLAYBACK and one for testing against Azure resources. This change here is updating the LIVE and RECORD profiles to use the configuration instead of being statically false as some service can easily transition their live testing to be parallel.